apps/prairielearn/src/lib/question.js

```diff
@@ -1299,14 +1330,40 @@ export async function startTestQuestion(
         course,
         type,
         authn_user_id,
-        (err, ret_success) => {
-          if (ERR(err, callback)) return;
-          success = success && ret_success;
-          callback(null);
-        },
       );
+      success = success && result.success;
+      if (stats) {
+        stats.push(result.stats);
+      }
     });
 
+    function printStats(label, key) {
+      let min = Number.MAX_SAFE_INTEGER;
+      let max = 0;
+      let count = 0;
+      let sum = 0;
```

Comment:
Are there any other stats folks want? I think this is a fine starting point.

Reply:
Yeah, I don't think you need more than that.